packages/components/src/Brand/Brand.tsx

```diff
@@ -58,11 +58,11 @@ export const Brand = ({
     )
   }
 
-  const { alt, classNameOverride } = restProps
+  const { alt, classNameOverride, ...otherProps } = restProps
   const brandTheme = reversed ? "-reversed" : "-default"
 
   return (
-    <picture className={classNameOverride} {...(restProps as PictureProps)}>
+    <picture className={classNameOverride} {...(otherProps as PictureProps)}>
```

Comment:
Either my IDE isn't showing me errors, otherwise this should be okay now :)

```suggestion
    <picture className={classNameOverride} {...otherProps}>
```